scripts/technical-debt-metrics.sh

```diff
@@ -81,7 +81,7 @@ for i in ${!titlesPathsAndRegexes[@]}; do
     then fl="-i"  # just for porting notes we ignore the case in the regex
     else fl="--"
     fi
-    printf '%s|%s\n' "$(git grep "${fl}" "${regex}" "${pathspec}" | wc -l)" "${title}"
+    printf '%s|%s\n' "$(git grep "${fl}" "${regex}" -- "${pathspec}" | wc -l)" "${title}"
```

Comment:
Can you please `s/test/MathlibTest/` on L71 as well?

Reply:
Thanks!  I also replaced a `test` in a comment with `MathlibTest`.